CODE_OF_CONDUCT.md

```diff
@@ -44,6 +44,6 @@ an expert in all the systems. Once you find the answer, document it in
 the first place you looked. That way, the next person will be brought
 up to speed even quicker.
 
-!["I try not to make fun of people for admitting they don't know things, because for each thing 'everyone knows' by the time they're adults, every day there are, on average, 10,000 people in the US hearing about it for the first time. If I make fun of people, I train them not to tell me when they have those moments. And I miss out on the fun." "Diet coke and mentos thing? What's that?" "Oh man! We're going to the grocery store." "Why?" "You're one of today's lucky 10,000."](https://imgs.xkcd.com/comics/ten_thousand.png)
+!["I try not to make fun of people for admitting they don't know things, because for each thing 'everyone knows' by the time they're adults, every day there are, on average, 10,000 people in the US hearing about it for the first time. If I make fun of people, I train them not to tell me when they have those moments. And I miss out on the fun." "Diet coke and mentos thing? What's that?" "Oh, man! We're going to the grocery store." "Why?" "You're one of today's lucky 10,000."](https://imgs.xkcd.com/comics/ten_thousand.png)
```

Comment:
I can't visually see the diff on this one...what was the change?

Reply:
It is the final space, highlight both and you will be able to see it 👀

It is my markdown linter, says it is best practices. if you think it is not necessary I will remove it as soon as possible

Reply:
also looks like you added a comma, not sure github diffing algorithm didn't pick it up, this LGTM

Reply:
Great! 👍